test/services/table-foreign-keys-analyzer.js

```diff
@@ -0,0 +1,31 @@
+const { expect } = require('chai');
+const SequelizeHelper = require('../utils/sequelize-helper');
+const { databases } = require('../utils/databases-urls');
+const TableForeignKeysAnalyzer = require('../../services/table-foreign-keys-analyzer');
+const expectedData = require('../expected/table-foreign-keys-analyzer-output');
+
+describe('Table analyser > SQL', () => {
```

Comment:
The file is name table-foreign-key-analyzer and here we have only table analyzer. Be consistent. The best reviewer should be yourself. Read your code again and again before submitting